### ISSUE TITLE
empty-line-between-blocks should support an option to allow one empty line only

### ISSUE THREAD

**Issue by joseluisq** [sasstools/sass-lint#433]
[empty-line-between-blocks](https://github.com/sasstools/sass-lint/blob/develop/docs/rules/empty-line-between-blocks.md) should support an option to **allow one empty line** between blocks.

For example:
```yaml
empty-line-between-blocks:
  - 1
  -
    include: true
    allow-single-line-rulesets: true
```

With above options, several empty lines are allowed:

![captura de pantalla de 2015-11-25 00-54-46](https://cloud.githubusercontent.com/assets/1700322/11389479/3294010e-930f-11e5-8b60-f197cb20e51d.png)

Adding `allow-one-empty-line: true` option:

```yaml
empty-line-between-blocks:
  - 1
  -
    include: true
    allow-single-line-rulesets: true
    allow-one-empty-line: true
```

![captura de pantalla de 2015-11-25 01-05-03](https://cloud.githubusercontent.com/assets/1700322/11389630/99eb22fa-9310-11e5-8c46-e68fde298af6.png)

Of course, this is a proposal option.
Regards.

**Comment by backflip** [comment]
I agree, this would be very helpful.

**Comment by Nirazul** [comment]
Maybe it's more useful to change it to:
`maximum-allowed-lines: n` 

So you can choose between allowing only one, allowing one or two, etc.

**Comment by backflip** [comment]
@Nirazul Even better.

**Comment by DanPurdy** [comment]
I think it makes sense, my only thoughts are how explicit this is. With an option like `maximum-allowed-lines` you would allow people to have say anywhere between 0 and `n` blank lines at any one time. Would it be better to enforce a value such as `required-empty-lines` or something?

**Comment by backflip** [comment]
A combination of `maximally-allowed-lines` and `minimally-required-lines` might fit most cases.

**Comment by Nirazul** [comment]
For my taste, it'd make more sense to change the rule like that:

1. Implement `allow-single-line-rulesets` as separate rule
2. Implement rule with a spacing option for pre and post block empty lines, something like [ESLint does](http://eslint.org/docs/rules/padded-blocks).

**Comment by joseluisq** [comment]
> Maybe it's more useful to change it to:
> maximum-allowed-lines: n
> 
> So you can choose between allowing only one, allowing one or two, etc.

@Nirazul It's a good idea, but I think 
What user would like to add more than one empty line among your properties?

**Comment by joseluisq** [comment]
Another consideration:
I don't know if `empty-line-between-blocks` and `allow-single-line-rulesets` have literally some relation. It could allow-single-line-rulesets (as @Nirazul says) be implemented separately.

**Comment by robertron0815** [comment]
+1
